src/components/Rating/Rating.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+function Rating(props) {
+  const {children} = props
+
+  const checkRating = (children) => {
```

Comment:
This function works but it's not very efficient. Other methods that would work would be better such as a loop or a string.repeat()